ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-ref.yaml

```diff
@@ -11,7 +11,7 @@ ref:
     mount_path: /secret
   env:
   - name: E2E_VERSION
-    default: "v2.1.4"
+    default: "v2.1.5"
```

Comment:
Do we want to be incrementing e2e version inconsistently across the workloads? cdv2, node-density, etc. will all be on v2.1.4 with this change.

Reply:
Oh. I see 2.1.5 updated consistently in the other PR.